focus-ios/Blockzilla/SearchSuggestions/OverlayView.swift

```diff
@@ -429,7 +429,7 @@ class OverlayView: UIView {
                 make.leading.trailing.equalTo(safeAreaLayoutGuide)
             }
             if let lastSearchButton = searchButtonGroup.last {
-                if searchButtonGroup.count >= 0 && !lastSearchButton.isHidden {
+                if searchButtonGroup.isEmpty && !lastSearchButton.isHidden {
```

Comment:
Here, the logic is also broken by using `isEmpty`. We also have the case if it is not empty. A solution could possibly be to assign a variable, e.g., `someVar = searchButtonGroup.count`, and use `someVar >= 0` to bypass the SwiftLint error.